code/modules/spells/spell_types/pointed/tie_shoes.dm

```diff
@@ -0,0 +1,130 @@
+	cooldown_reduction_per_rank = 0.2 SECONDS
+
+	spell_max_level = 4
+	invocation = "Acetato!"
+	invocation_type = INVOCATION_SHOUT
+	spell_requirements = NONE
+	antimagic_flags = MAGIC_RESISTANCE|MAGIC_RESISTANCE_HOLY
+
+	cast_range = INFINITY
+	active_msg = "You prepare to tie your target's shoes!"
+
+	/// Ignores inability to tie laces, such as jackboots, magboots, or sandals.
+	var/bypass_tie_status = FALSE
+	/// Summons shoes to untie if the target has none.
+	var/summons_shoes = FALSE
+
+/datum/action/cooldown/spell/pointed/untie_shoes/level_spell(bypass_cap)
+	. = ..()
+	if(spell_level == 2)
+		bypass_tie_status = TRUE
+		to_chat(owner, span_notice("You will now summon laces on laceless shoes, such as jackboots."))
+
+	if(spell_level == 3)
+		summons_shoes = TRUE
+		to_chat(owner, span_notice("You will now summon shoes if your target has none."))
+
+	if(spell_level == 4)
+		invocation_type = INVOCATION_NONE
+		to_chat(owner, span_boldnotice("Your invocations are now silent!"))
+
+/datum/action/cooldown/spell/pointed/untie_shoes/is_valid_target(atom/cast_on)
+	return isliving(cast_on)
+
+// We need to override this, as trying to change next_use_time in cast() will just result in it being overridden.
+/datum/action/cooldown/spell/touch/before_cast(atom/cast_on)
+	return ..() | SPELL_NO_IMMEDIATE_COOLDOWN
+
+/datum/action/cooldown/spell/pointed/untie_shoes/cast(mob/living/carbon/cast_on)
+	. = ..()
+	if(cast_on.can_block_magic(antimagic_flags))
+		to_chat(owner, span_warning("The spell had no effect!"))
+		return FALSE
+
+	if(isanimal_or_basicmob(cast_on))
+		cast_on.add_movespeed_modifier(/datum/movespeed_modifier/magic_ties)
+		addtimer(CALLBACK(cast_on, TYPE_PROC_REF(/mob/living, remove_movespeed_modifier), /datum/movespeed_modifier/magic_ties), 3 SECONDS * spell_level, TIMER_UNIQUE|TIMER_OVERRIDE)
+		to_chat(owner, span_warning("You tie [cast_on] with weak, magic laces!"))
+		if(invocation_type != INVOCATION_NONE) // extra feedback since it's weird for them
+			cast_on.balloon_alert_to_viewers("magically tied!")
+		else
+			cast_on.balloon_alert(owner, "magically tied!")
+		playsound(cast_on, 'sound/magic/summonitems_generic.ogg', 50, TRUE)
+		return TRUE
+
+	var/shoe_to_cast = /obj/item/clothing/shoes/sneakers/random
+
+	if(HAS_TRAIT(owner, TRAIT_CHUUNIBYOU))
+		shoe_to_cast = /obj/item/clothing/shoes/sneakers/marisa
+	if(HAS_TRAIT(owner, TRAIT_SPLATTERCASTER))
+		shoe_to_cast = /obj/item/clothing/shoes/laceup
+
+	var/obj/item/clothing/shoes/shoes_to_tie = cast_on.shoes
+
+	if(isnull(shoes_to_tie))
+		if(summons_shoes)
+			shoes_to_tie = new shoe_to_cast(cast_on)
+			if(!cast_on.equip_to_slot_or_del(shoes_to_tie, ITEM_SLOT_FEET))
+				to_chat(owner, span_warning("Couldn't equip shoes on [cast_on]!"))
+				return FALSE
+			else if(invocation_type != INVOCATION_NONE)
+				playsound(cast_on, 'sound/magic/summonitems_generic.ogg', 50, TRUE)
+			else
+				to_chat(owner, span_warning("[cast_on] isn't wearing any shoes!"))
+			return FALSE
```

Comment:
i think you're going about this the wrong way, shouldn't it be:

```dm
if(isnull(shoes_to_tie)
 if(!summon_shoes)
  to_chat(owner, span_warning("[cast_on] isn't wearing any shoes!"))
  return FALSE

 shoes_to_tie = new shoe_to_cast(cast_on)
 if(!cast_on.equip_to_slot_or_del(shoes_to_tie, ITEM_SLOT_FEET))
  to_chat(owner, span_warning("Couldn't equip shoes on [cast_on]!"))
  return FALSE

 if(invocation_type != INVOCATION_NONE)
  playsound(...)

// rest of code continues from here as normal
```

Reply:
i was considering it, but it already unties them first so that would mean itd be 3 casts to knot which im iffy about